Fika.Core/Coop/GameMode/CoopGame.cs

```diff
@@ -1714,6 +1730,30 @@ public override void Stop(string profileId, ExitStatus exitStatus, string exitNa
             GClass549.Config.UseSpiritPlayer = false;
         }
 
+        private void SavePlayer(CoopPlayer player, ExitStatus exitStatus, string exitName)
+        {
+            //Since we're bypassing saving on exiting, run this now.
+            player.Profile.EftStats.LastPlayerState = null;
+            player.StatisticsManager.EndStatisticsSession(exitStatus, base.PastTime);
+            player.CheckAndResetControllers(exitStatus, base.PastTime, base.Location_0.Id, exitName);
+
+            //Method taken directly from AKI, can be found in the aki-singleplayer assembly as OfflineSaveProfilePatch
+            var converterClass = typeof(AbstractGame).Assembly.GetTypes().First(t => t.GetField("Converters", BindingFlags.Static | BindingFlags.Public) != null);
```

Comment:
Do you mind using explicit type instead of `var`? Either way works, but I prefer explicit, so not a biggie.